CoreScriptsRoot/Modules/BackpackScript3D.lua

```diff
@@ -219,6 +244,19 @@ local function AddTool(tool)
 	slot.text.ClipsDescendants = true
 	slot.text.Selectable = false
 
+	local selectionObject = Util:Create'ImageLabel'
+	{
+		Name = 'SelectionObject';
+		Size = UDim2.new(1,0,1,0);
+		BackgroundTransparency = 1;
+		Image = "rbxasset://textures/ui/Keyboard/key_selection_9slice.png";
```

Comment:
Just a heads up Miguel has an updated version of this nine-slice with different thicknesses